sql/driver/wire.go

```diff
@@ -80,14 +100,17 @@ func (d Datum) Value() (driver.Value, error) {
 		val = t.BytesVal
 	case *Datum_StringVal:
 		val = t.StringVal
+	case *Datum_DateVal:
+		val = Date{t.DateVal.GoTime()}
```

Comment:
Reading this again, won't `select now()::date, now()` also print the date in the local time zone? I think the result will be ('2015-09-20', '2015-09-20 20:00:00').